resources/icons/NorwegianSignals/Large/readme.txt

```diff
@@ -0,0 +1,3 @@
+Started with something from
+C:\Program Files (x86)\JMRI\resources\icons\largeschematics\aspects\CSD-1962
```

Comment:
Maybe this link should be changed to `resources\icons\largeschematics\aspects\CSD-1962`